apps/osvr_server.cpp

```diff
@@ -106,36 +98,29 @@ int main(int argc, char *argv[]) {
         log->debug("Verbose logging enabled.");
     }
 
-    configName = values["config"].as<std::string>();
+    if (values.count("config")) {
+        std::string configFileArgument = values["config"].as<std::string>();
+        log->info() << "Using config file " << configFileArgument << " from command line argument.";
+        configPaths = { configFileArgument };
+    } else {
+        log->info() << "Using default config files - pass a filename on the command "
+            "line to use a different one.";
+        configPaths = osvr::server::getDefaultConfigFilePaths();
+    }
 
-    boost::optional<fs::path> configPath(configName);
-    try {
-        if (!fs::exists(*configPath)) {
-            log->warn() << "File '" << configName
-                << "' not found.  Using empty configuration.";
-            configPath = boost::none;
+    server = osvr::server::configureServerFromFirstFileInList(configPaths);
+    if (!server) {        
+        // only attempt to load the empty config if no arguments are passed.
+        if (!values.count("config")) {
+            log->info() << "Could not find a valid config file. Using default config object.";
```

Comment:
Is this maybe better phrased "could not find config file with default name" or something like that? The user seeing this message wouldn't have passed a config file, so they might be confused as to why they're seeing this.